### PR TITLE
Topic 04 excercise 03

### PR DESCRIPTION
**Pull request by vvale93**

### CODE REVIEW

File: Topic-04-Excercise-03/script.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+function sendMsg(){
+    if ('WebSocket' in window){
+        console.log('WebSocket is supported by your Browser!');
+        let ws = new WebSocket('ws://demos.kaazing.com/echo');
+        let msg = document.getElementById('inputMsg').value;
+        
+        if(!msg){
+            document.getElementById('responseList').innerHTML += `<li>You didn\'t write a message!</li>`;
```
Comment (DemianMoschin-GLB) [inline Topic-04-Excercise-03/script.js]: Template literals for this. Don't use ```+=```

Comment (DemianMoschin-GLB) [reply Topic-04-Excercise-03/script.js]: Review the rest of your script. You're using this pattern everywhere

File: Topic-04-Excercise-03/script.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+       
+        ws.onclose = function(){ 
+            console.log('Connection is closed'); 
+        };
+           
+        window.onbeforeunload = function(e){
+            socket.close();
+        };
+
+    }else{
```
Comment (DemianMoschin-GLB) [inline Topic-04-Excercise-03/script.js]: Please include spaces between braces.

File: Topic-04-Excercise-03/script.js (hunk 1 above)
Comment (vvale93) [reply Topic-04-Excercise-03/script.js]: changed

File: Topic-04-Excercise-03/script.js (hunk 2 above)
Comment (vvale93) [reply Topic-04-Excercise-03/script.js]: done

**Pull request merged**